<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constraint Properties</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="vectorPotentialLbl">
     <property name="text">
      <string>Vector Potential:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Gui::InputField" name="vectorPotentialTxt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="singleStep" stdset="0">
        <double>1.000000000000000</double>
       </property>
       <property name="unit" stdset="0">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="vectorPotentialBox">
       <property name="text">
        <string>unspecified</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="vectorPotentialConstantBox">
     <property name="text">
      <string>Vector Potential Constant</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="magneticInfinityBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Farfield / Magnetic infinity</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>vectorPotentialBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>vectorPotentialTxt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vectorPotentialBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>vectorPotentialTxt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
